### DISCUSSION TITLE
importing the dependencies

### DISCUSSION THREAD

**Discussion by Nambunga** [IBAHCM/RPiR#292] [Archive] [answered]
@soniamitchell Hello! I have a question for clarification. If you develop a package and at some point you need to use some dependencies such as dplyr, gigplot2 or others, then must these dependencies be put into your description file in your created package? Or if some dependencies are built in base R then there is no need to put it in your description file as will be automatically loaded?
Cheers

**Comment by soniamitchell** [comment, marked as answer]
Hello! If you’re adding dependencies you should add your dependency like this https://ibahcm.github.io/RPiR/articles/pages/packages_guide.html#add-dependencies base R functions don’t need to be added 👍

**Reply by Nambunga** [reply]
Thanks 👍
